.github/workflows/lte-integ-test-containerized.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         run: cat lte/gateway/docker/docker-compose.yaml
       - name: Run the integration test
         env:
-          DOCKER_REGISTRY: docker-ci.artifactory.magmacore.org/
+          DOCKER_REGISTRY: linuxfoundation.jfrog.io/magma-docker-test/
```

Comment:
FYI: This has a conflict with #14534 where I used `inputs.registry` and inlined it into the `fab` command below.

Reply:
I will rebase after that one is merged.